supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -72,7 +72,10 @@ Warn users about the following deprecation stages:
 * Plan to remove
 * Remove
 
-If available, inform users of alternative capabilities and workarounds.
+If available, inform users of alternative capabilities and workarounds. 
+
+If necessary, you can also refer to a specific release timeline when the deprecation or removal will occur, even though our general rule is to avoid making statements that predict future releases or release dates.
```

Comment:
This is a solid approach. To be more succinct and stress the importance of caution, consider the following revision:
```suggestion
Avoid predictions about specific releases or release dates.
If necessary, you can mention current plans to deprecate or remove a capability in a specific release. Plans can change.
```

Reply:
+1 this is a great suggestion

Reply:
Thanks, Rolfe and Sara. Made most of the changes, but left off the last suggested sentence because I think it states the obvious.